sdk/src/main/java/com/mapbox/maps/StyleObserver.kt

```diff
@@ -20,9 +20,13 @@ internal class StyleObserver(
   private val pixelRatio: Float
 ) : OnStyleLoadedListener, OnMapLoadErrorListener, OnStyleDataLoadedListener {
 
-  private var loadStyleListener: Style.OnStyleLoaded? = null
+  private var userStyleLoadedListener: Style.OnStyleLoaded? = null
```

Comment:
Not sure we need `user` prefix. I'd use previous `loadStyleListener` naming to make delta less

Reply:
Previously it's really confusing since we also have `val styleLoadedListener: Style.OnStyleLoaded,` as a property, then another property  with different word order inside `loadStyleListener: Style.OnStyleLoaded`